tensorflow/BUILD

```diff
@@ -539,6 +554,93 @@ tf_cc_shared_object(
     ] + if_ngraph(["@ngraph_tf//:ngraph_tf"]),
 )
 
+# ** Targets for Windows build (start) **
```

Comment:
I claim no understanding of what's happening in this block :)

That said, would it make sense (especially if all these rules are for Windows only) to prefix the name of these targets with `windows_`? (e.g. `windows_tf_custom_op_library_additional_deps.dll` etc.)

@allenlavoie @gunan in case they have a better understanding of shared libraries in Windows.
Otherwise LGTM :)

Reply:
While that does make sense, but I'm a bit unwilling to make the target name longer. Because we often have some issue when dealing with long path on Windows. In C++ compilation, the compiler tool won't allow a file path to be longer than 260 characters. 

I think users should know it's a windows specific target when they see `dll` or `import library`, etc.